src/services/store.js

```diff
@@ -17,7 +18,9 @@ switch (process.env.NODE_ENV) {
     store = createStore(rootReducer, applyMiddleware(...middleware))
     break
   default:
-    middleware.push(createLogger())
+    if (urlParams.has('logger')) {
```

Comment:
This is looking at `process.env.NODE_ENV` to check on whether to import the logger, so I don't think we need an additional env setting, but I can see if there is another way to actually toggle it on and off in the console without needing a query param. Do you have an example of using dev tools?

Reply:
If I may be so bold as to chime in...  Since the logger is only loaded if `NODE_ENV` is not production or test, a query parameter to control it doesn't seem so much like a hack, and it's nice to be able to toggle debug tools on and off easily.

Doing it strictly with an environment variable would require either tweaking the webpack config to read from the `.env` file or restarting the `js` container to get the updated env vars.  One is fragile (webpack configs, amirite?) and the other is cumbersome and slow.

Another approach would be a button that only shows up in dev that sets some dev-only redux state, but that wouldn't work because by then, the store is already created and configured.  😄

The only caution I'd have here is to defaulting to disabled - but that's a question kind of about the maturity of the app.  If you're not having to debug reactions very often, then defaulted to disabled might make a lot of sense.  That's definitely something y'all would know more about, but just throwing that out there.

Reply:
@mgwalker @ryanhofdotgov updated to be on by default